source/guides/maintenance/developer-workstation.rst

```diff
@@ -17,67 +17,204 @@ other Linux\* distributions.
-
-   * - :command:`dev-utils`
-     -
-     -
-     - ✓
+After installing the minimum set of bundles required to get started, you can 
+add more bundles relevant to your specific use case.
+
+To run any process required for |CL| development, you can add the large 
+bundle :ref:`*os-clr-on-clr* <enable-user-space>`. However, you may want to deploy a leaner OS with only bundles relevant to your project.
+
+Use the **Developer Profiles** tabs to start installing *suggested bundles* based on your role or project. Consider these profiles as a starting point.
+
+..tip:: Add description on click on bundle to learn.... 
+
+
+.. tabs::
+
+   .. tab:: AI/ML Engineer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundles
+
+         * - Build machine learning applications with a full suite of libraries.
+           - `machine-learning-basic <https://clearlinux.org/software/bundle/machine-learning-basic/>`_
+
+         * - Build machine learning applications with PyTorch, an optimized tensor library for deep learning.
+           - `machine-learning-pytorch <https://clearlinux.org/software/bundle/machine-learning-pytorch/>`_
+
+         * - Build machine learning applications using Tensorflow, a library for numerical computation using deep neural networks.
+           - `machine-learning-tensorflow <https://clearlinux.org/software/bundle/machine-learning-tensorflow/>`_
+
+         * - Web-based, interactive tools for machine learning.
+           - `machine-learning-web-ui <https://clearlinux.org/software/bundle/machine-learning-web-ui/>`_
+
+         * - Machine learning Docker container.
+           - `machine-learning <https://clearlinux.org/software/docker/machine-learning/>`_
+
+         * - Pre-built Python libraries for Data Science.
+           - `python-extras <https://clearlinux.org/software/bundle/python-extras>`_
+
+         * - API helper for cloud access.
+           - `cloud-api <https://clearlinux.org/software/bundle/cloud-api/>`_
+
+   .. tab:: Computer Vision Engineer
+
+      .. list-table:: 
+         :widths: 50, 50
+         :header-rows: 1
+
+         * - Function
+           - Bundles
+
+         * - Build computer vision applications.
+           - `computer-vision-basic <https://clearlinux.org/software/bundle/computer-vision-basic/>`_
+
+         * - Work with deep learning and edge-optimized models.
+           - `computer-vision-models <https://clearlinux.org/software/bundle/computer-vision-models/>`_
+
+         * - Basic OpenVINO™ toolkit.
+           - `computer-vision-openvino <https://clearlinux.org/software/bundle/computer-vision-openvino/>`_
+
+         * - API helper for cloud access.
+           - `cloud-api <https://clearlinux.org/software/bundle/cloud-api/>`_
+
+         * - All content for pkgconfig file opencv.pc.
+           - `devpkg-opencv <https://clearlinux.org/software/bundle/devpkg-opencv/>`_
+
+   .. tab:: Cloud Orchestration Engineer
```

Comment:
@CraigSterrett Could you please review and advise on the bundles I included for `Cloud Orchestration Engineer`? 
My goal is to list at least 3 bundles per profile. Suggest removals too as appropriate.